### ISSUE TITLE
Rounded svgs are not smooth in Impeller

### ISSUE THREAD

**Issue by mattermoran** [dnfield/flutter_svg#830]
I know impeller is still in beta but still wanted to check if this is something that will just get fixed later by impeller or some work needed to be done in flutter_svg?

|                                Marker                                 |                                Navigation                                 |
| :-------------------------------------------------------------------: | :-----------------------------------------------------------------------: |
| ![Skia](https://user-images.githubusercontent.com/29712386/214808202-9e031d19-6ba4-4db8-8230-17162527c9ba.png) | ![Impeller](https://user-images.githubusercontent.com/29712386/214808220-62ca686b-fbe6-4138-acb3-05aee07e1b52.png) |

flutter_svg 2.0.0-dev.2

<details>
  <summary>flutter doctor output</summary>
  
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.7.0, on macOS 13.1 22C65 darwin-arm64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.2)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.3)
[✓] Android Studio (version 2021.3)
[✓] VS Code (version 1.74.3)
[✓] Connected device (3 available)
[✓] HTTP Host Availability

• No issues found!
```
</details>

<details>
<summary>svgs</summary>

![search](https://user-images.githubusercontent.com/29712386/214809711-c76eb85c-44a3-47db-9c1a-ae5d7258f969.svg)
![heart_outline](https://user-images.githubusercontent.com/29712386/214809751-fe867103-ccd9-41f7-82eb-5c25317050bf.svg)
![settings](https://user-images.githubusercontent.com/29712386/214809723-c2fa85b5-a64c-4592-ae40-dac8a0268c2e.svg)
</details>

**Comment by dnfield** [comment]
Does this look any better on flutter_svg 1.x?

**Comment by dnfield** [comment]
Also, can you share some application code that reproduces this? I'mwondering if there's any transform going on.

What's likely is that the tolerance values used to flatten the cubics are off.

**Comment by mattermoran** [comment]
v1 seem to have the same issue. 

<details>

<summary>code to reproduce</summary>

```dart
Scaffold(
  body: Column(
    mainAxisAlignment: MainAxisAlignment.center,
    crossAxisAlignment: CrossAxisAlignment.stretch,
    children: [
      SvgPicture.asset(
        'assets/svgs/search.svg',
        width: 200,
      ),
      const SizedBox(height: 16),
      SvgPicture.asset(
        'assets/svgs/settings.svg',
        width: 200,
      ),
      const SizedBox(height: 16),
      SvgPicture.asset(
        'assets/svgs/heart_outline.svg',
        width: 200,
      ),
    ],
  ),
)
```

</details>

**Comment by Kiruel** [comment]
Same problem on `flutter_svg: ^2.0.2`
Flutter version: `3.7.3`

**Comment by dnfield** [comment]
This is a bug in impeller. Looking at it now.

**Comment by dnfield** [comment]
Closing this in favor of https://github.com/flutter/flutter/issues/121540

**Comment by mathewthe2** [comment]
Is this issue resolved on the master channel?

**Comment by dnfield** [comment]
Yes.

**Comment by armandojimenez** [comment]
is this available in FLutter 3.10? My some of my SVGs looks jagged or incomplete in Impeller

**Comment by dnfield** [comment]
> is this available in FLutter 3.10? My some of my SVGs looks jagged or incomplete in Impeller

File bugs with reproductions please
